pbsmetrics/metrics.go

```diff
@@ -314,6 +314,7 @@ type MetricsEngine interface {
 	RecordUserIDSet(userLabels UserLabels) // Function should verify bidder values
 	RecordStoredReqCacheResult(cacheResult CacheResult, inc int)
 	RecordStoredImpCacheResult(cacheResult CacheResult, inc int)
+	RecordAccountCacheResult(cacheResult CacheResult, inc int)
```

Comment:
The `inc` parameter seems to always be 1. One on hand it's nice to keep symmetry with the other CacheResult methods, but those pass a computed value. Thoughts?

Reply:
I felt I should leave open the possibility of fetching multiple accounts if ever need to, as the metric update does not lose much in readability.

I don't have a strong preference other than this having been easy to write and test given the symmetry :)

Reply:
Neither do I. Let's leave it as for the sake of symmetry.